docs/networks/testnet/become-a-validator.md

```diff
@@ -33,15 +36,58 @@ Please refer to the regular [Node Guide](./running-a-node.md) that explains the
 
 Set up your regular node using the LUKSO CLI as described in the [Node Guide](./running-a-node.md).
 
-The instructions are the same as for mainnet node, please refer to [mainnet validator guide](../mainnet/become-a-validator.md#using-lukso-cli). You will simply have to add `--testnet` after each commands to make it work for testnet. The command will look like this:
+The instructions are the same as for mainnet node, please refer to [Mainnet Validator Guide](../mainnet/become-a-validator.md#using-lukso-cli). You will simply have to add the `--testnet` flag to each commands. The commands will look like this:
+
+#### Importing Validator Keys
 
 ```bash
 $ lukso validator import --validator-keys "./path/to/your/keys/folder" --testnet
 
 $ lukso validator list --testnet
+```
+
+#### Starting the Validator
+
+Without specifying any flags, the node starts its normal synchronization process.
+
+If you want more convenience and your validator to operate quickly, you can also use checkpoints. Checkpoint synchronization is a feature that significantly speeds up the initial sync time of the consensus client. If enabled, your node will begin syncing from a recently finalized consensus checkpoint instead of genesis.
+
+<Tabs>
+  <TabItem value="regular-sync" label="Regular Synchronization">
+
+:::info
+
+The synchronization process will take multiple hours for the validator to participate in the consensus.
+
+:::
+
+```bash
+$ lukso start --testnet --validator --transaction-fee-recipient "0x1234..."
+```
 
-$ lukso start --validator --transaction-fee-recipient "0x1234..." --testnet
+  </TabItem>
+    <TabItem value="checkpoint-sync" label="Checkpoint Synchronization">
 
+:::info
+
+The shortcut is ideal for making installation, validator migration, or recovery much faster.
+
+:::
+
+```sh
+# Testnet Checkpoint for Prysm Consensus Client
+$ lukso start --testnet --validator --transaction-fee-recipient "0x1234..." --prysm-checkpoint-sync-url=https://checkpoints.testnet.lukso.network
+
+# Testnet Checkpoint for Lighthouse Consensus Client
+$ lukso start --testnet --validator --transaction-fee-recipient "0x1234..." --lighthouse-checkpoint-sync-url=https://checkpoints.testnet.lukso.network
+```
+
+  </TabItem>
+</Tabs>
```

Comment:
maybe to avboid duplicating content you can see: check the mainnet page and use `https://checkpoints.testnet.lukso.network` instead of `https://checkpoints.mainnet.lukso.network`